src/Stratis.Bitcoin.Features.BlockStore/BlockStoreQueue.cs

```diff
@@ -300,7 +300,7 @@ public void AddToPending(ChainedHeaderBlock chainedHeaderBlock)
 
                     this.currentBatchSizeBytes += item.Block.BlockSize.Value;
 
-                    saveBatch = saveBatch || (item.ChainedHeader == this.chain.Tip) || (this.currentBatchSizeBytes >= BatchThresholdSizeBytes);
+                    saveBatch = saveBatch || (this.currentBatchSizeBytes >= BatchThresholdSizeBytes);
```

Comment:
I'm not sure this is correct

Reply:
Why? 
My arguments for this change are in the slack.